pages/docs/reference/coding-conventions.md

```diff
@@ -726,6 +727,195 @@ foo {
 
 </div>
 
+### Trailing commas
+
+Use trailing commas at declaration site, including multi-line enumerations such as argument and parameter lists, `when` entries, and components of destructuring declarations.
```

Comment:
I'm not sure that readers are familiar with the term "trailing comma". Perhaps we should add a definition.